### PR TITLE
docstring updated to indicate limits of axis_tilt

### PR DESCRIPTION
**Pull request by kurt-rhee**
- [Y] Adds docstring information from discussion in #1471 
 - [Y] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [N] Tests added
 - [N] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/reference) for API changes.
 - [N] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [Y] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [Y] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

### CODE REVIEW

**Comment by kurt-rhee** [comment]
@kanderso-nrel Added those additional references when axis_tilt is used as an input and also added myself to the list of contributors.  Thanks for the help!

File: pvlib/tracking.py [hunk 1]
```diff
@@ -524,7 +527,10 @@ def calc_surface_orientation(tracker_theta, axis_tilt=0, axis_azimuth=0):
 def calc_axis_tilt(slope_azimuth, slope_tilt, axis_azimuth):
     """
     Calculate tracker axis tilt in the global reference frame when on a sloped
-    plane.
+    plane. Tracker axis tilt is a limited positive angle equaling a minimum of
+    0 when the normal vector of the module's surface points straight skyward,
+    parallel to the z-axis and a maximum of 90 when the normal vector of the
+    modules surface is parallel with the y-axis.
```
[the remark this answers is not in the document]
Comment (kurt-rhee) [reply pvlib/tracking.py:533]: Much better than my attempt, I'll make the commit now.

File: pvlib/tracking.py [hunk 2]
```diff
@@ -295,6 +296,7 @@ def singleaxis(apparent_zenith, apparent_azimuth,
     axis_tilt : float, default 0
         The tilt of the axis of rotation (i.e, the y-axis defined by
         axis_azimuth) with respect to horizontal, in decimal degrees.
+        axis_tilt must be >= 0 and <= 90.
```
Comment (cwhanse) [inline pvlib/tracking.py:299]:
```suggestion
        axis_azimuth) with respect to horizontal.
        axis_tilt must be >= 0 and <= 90. [degree]
```

File: pvlib/tracking.py [hunk 3]
```diff
@@ -639,7 +644,8 @@ def calc_cross_axis_tilt(
     axis_azimuth : float
         direction of tracker axes projected on the horizontal [degrees]
     axis_tilt : float
-        tilt of trackers relative to horizontal [degrees]
+        tilt of trackers relative to horizontal.  axis_tilt must be >= 0
+        and <= 90.[degrees]
```
Comment (cwhanse) [inline pvlib/tracking.py:648]:
```suggestion
        tilt of trackers relative to horizontal.  ``axis_tilt`` must be >= 0
        and <= 90. [degree]
```

**Pull request merged**